### PR TITLE
luci-app-strongswan-ipsec: add package

### PR DESCRIPTION
**Pull request by mips171**
I'm not an IPSec pro, so if there's anything missing or wrong please point it out and I'll be sure to address it. Should we include the example config in the LuCI package? I'm thinking no, it should be a part of the strongswan package. Thoughts?

### CODE REVIEW

**Comment by systemcrash** [comment]
The two packages strongswan and libreswan are largely synonymous, with slight behaviour differences which should not affect most users day to day. 

[I made the comment on the libreswan package effort](https://github.com/openwrt/luci/pull/5932#issuecomment-1324416438) a while back whether it would do as well to have strongswan also covered there. I don't actively use SS or LS so it's not in my purview to test it (wireguard floats my boat fine).

Something you could check? One GUI to rule them all...

https://github.com/openwrt/luci/pull/5932

**Comment by lvoegl** [comment]
What do you think of renaming the package to **luci-app-strongswan** since it generates uci configurations for both `strongswan-ipsec` and `strongswan-swanctl`?

File: applications/luci-app-strongswan-ipsec/root/usr/share/luci/menu.d/luci-app-strongswan-ipsec.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"admin/vpn/strongswan-ipsec": {
+		"title": "Strongswan IPSec",
+		"order": 90,
+		"action": {
+			"type": "view",
+			"path": "strongswan-ipsec"
```
[the remark this answers is not in the document]
Comment (systemcrash) [reply applications/luci-app-strongswan-ipsec/root/usr/share/luci/menu.d/luci-app-strongswan-ipsec.json:7]: should or must? 😉

**Comment by systemcrash** [comment]
This luci package also needs special treatment as @lvoegl did in #6700 

See #6700 for inspiration as to how to mark cryptos as unsafe.

**Comment by feckert** [comment]
Do we still need this PR? @lvogel has taken this PR and extended and changed it, so that it generates the strongswan config for swanctl backend. The question is now do we still need or want to support via LuCI application the old IPsec strongswan config backend?The old backend is deprecated and we want to use the new one via swanctl.

**Comment by mips171** [comment]
No, if it's deprecated so vote to close this one.

**Comment by mips171** [comment]
Sorry I misread earlier. I vote no don't support deprecated features in new software.

**Comment by feckert** [comment]
@systemcrash Why did you merged that?
This is the old ipsec handling and not the new swanctl handling. I thought we wanted to support the new one?
Your already merged the new swanctl handing https://github.com/openwrt/luci/pull/6700

**Comment by systemcrash** [comment]
I did not merge this PR.

The master tree shows that I merged #6700. 

This showing as 'merged' has to do with the fact that the *first* commit from #6700 uses this commit. 

Weird huh?

**Comment by feckert** [comment]
This is spooky. But good to know. Then forget my comment :-)

**Pull request merged**